src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/TextAreaComponent.tsx

```diff
@@ -1,22 +1,20 @@
 import * as React from 'react';
+import { IComponentProps } from '..';
 import '../../styles/shared.css';
 
-export interface ITextAreaComponentProps {
-  id: string;
-  formData: any;
-  handleDataChange: (value: any) => void;
-  isValid?: boolean;
-  readOnly: boolean;
-}
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface ITextAreaComponentProps extends IComponentProps {}
```

Comment:
Would prefer to just use the IComponentProps, and extend once the need arises. No big issue, just feel these disable lines are annoying

Reply:
I could not decide what was best, so I ended up doing differently in TextArea and Paragraph. I guess I'll fix this one then.

Reply:
`ITextAareaComponentProps` is exported and used in the tests. Seems rather strange to use `IComponentProps` in the tests?

What to do?
```suggestion
export type ITextAreaComponentProps = IComponentProps
```

Reply:
We could also add

```
{
  "@typescript-eslint/no-empty-interface": [
    "error",
    {
      "allowSingleExtends": false
    }
  ]
}
```

https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-empty-interface.md#options

Reply:
@ivarne why does it seem strange to use `IComponentProps `in the tests? Seems fine imo.

Reply:
I just like the pattern of all props being named `ComponentNameProps`. It just seems more correct and future proof, because a future extention would require fewer lines of code to change, and make it easier to search for usages.

Why do you dislike `allowSingleExtends`?

Reply:
Honestly, I have no strong opinion and do see your point 🤔 If you want to disable the rule I'm comfortable with that too. 

Either way, great work 👏